taskboot/docker.py

```diff
@@ -245,3 +256,37 @@ def _find_replacement(original):
     parser.content = open(dockerfile).read()
     logger.info('Initial parent images: {}'.format(' & '.join(parser.parent_images)))
     parser.parent_images = list(map(_find_replacement, parser.parent_images))
+
+
+def read_manifest(path):
+    '''
+    Read a Docker archive manifest and load it as JSON
+    '''
+    assert os.path.exists(path), 'Missing archive {}'.format(path)
+    assert tarfile.is_tarfile(path), 'Not a TAR archive {}'.format(path)
+
+    tar = tarfile.open(path)
```

Comment:
Nit: you could use a context manager here so you ensure the tar file is closed after.